### ISSUE TITLE
Back End Tech

### ISSUE THREAD

**Issue by taylormeyer237** [taylormeyer237/Drinking-Buddies#2]
What database do we use?
-SQL?
-noSQL?
if going on the play store i believe that is were it is hosted, do we need deploy,ent elsewhere?

**Comment by samualmoot** [comment]
MongoDB, CouchDB, CouchBase, Cassandra, HBase, Redis, Riak, Neo4J are the popular NoSQL databases examples.

I could use a lot of brushing up here. I think we've both used MongoDB before. I see Redis used a bit, might look into that. Ultimately something cheap?

**Comment by taylormeyer237** [comment]
yes ultimately cheap and on the applications requirements are the thing that I would prefer.
CouchDB, Cassandra, HBase, Redis, Riak, Neo4J I personally havn't seen but again this is something for us both to sharpen and grow so if you see them we can look into them.


As per mongoDB we both used at OPSpark but I haven't really utilized a DB in over a year as it hasn't been needed in what i was doing so this is something I am open to major discussion about.
